src/ssl.c

```diff
@@ -16623,6 +16623,62 @@ int wolfSSL_CTX_set_max_proto_version(WOLFSSL_CTX* ctx, int ver)
     return sanityCheckProtoVersion(ctx);
 }
 
+WOLFSSL_API int wolfSSL_CTX_get_min_proto_version(WOLFSSL_CTX* ctx)
```

Comment:
This essentially does the reverse of the mapping in wolfSSL_CTX_set_min_proto_version.